packages/core/src/components/textarea/textarea.story.tsx

```diff
@@ -0,0 +1,21 @@
+import { TextareaProps as BaseProps } from '@onfido/castor';
+import { html } from '../../../../../docs';
+import { c, classy, m } from '../../utils';
+
+export interface TextareaProps extends BaseProps {
+  rows?: number;
+  value?: string;
+}
+
+export const Textarea = ({
+  resize = 'vertical',
+  rows = 3,
+  invalid,
+  ...props
+}: TextareaProps) =>
+  html('textarea', {
+    ...props,
+    class: classy(c('textarea'), m({ invalid })),
+    resize,
```

Comment:
Currently does nothing, needs to be `style`

Reply:
Good catch, a little too late... Will open a fix.